### PR TITLE
ui. add alias description near hostname field

### PR DESCRIPTION
**Pull request by federicoballarini**
Refer to https://github.com/NethServer/dev/issues/5990

### CODE REVIEW

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -167,7 +167,8 @@
     "shutdown_ok": "The system is shutting down...",
     "shutdown_error": "Shutting down error occured.",
     "hostname_change": "After hostname change you have to login again",
-    "hostname_invalid_warn": "The server FQDN must not contain 'localhost' or 'localdomain'. You have to change your machine hostname before proceeding with other configurations"
+    "hostname_invalid_warn": "The server FQDN must not contain 'localhost' or 'localdomain'. You have to change your machine hostname before proceeding with other configurations",
```
Comment (edospadoni) [inline ui/public/i18n/language.json]: I can't see the difference, please dot not add this on the patch

Comment (federicoballarini) [reply ui/public/i18n/language.json]: I've added alias string (removed from other PR). It's not correct?

Comment (edospadoni) [reply ui/public/i18n/language.json]: The `alias` translation is already inserted in the file: https://github.com/NethServer/nethserver-cockpit/blob/725b01d6b6234bc38575c8cfba3e83705f7b9e66/ui/public/i18n/language.json#L139 It's a duplicated string

Comment (federicoballarini) [reply ui/public/i18n/language.json]: Ops sorry. I didn't see it. I will remove it in a minute.

**Pull request merged**